docs/reference/nodes.md

```diff
@@ -35,13 +40,13 @@ Plugs define the input and output data of the nodes. Data values can be set thro
 | Type | Description | Compatible Python Type |
 | :--- | :---------- | :---|
 | *Bool* | Defines a boolean value.|`Bool`|
-| *Code* | Defines a python code block. This type is used whenever custom scripting is required in a node. A plug of this type displays as an embedded text editor in the inspector. |`String`|
+| *Code* | Defines a Python code block. This type is used whenever custom scripting is required in a node. A plug of this type displays as an embedded text editor in the inspector. |`String`|
 | *Color* | Defines a color in RGBA. Interacting with a plug of this type opens a color picker dialog.|`List`|
 | *Dict* | Defines a dictionary value.|`Dict`|
 | *Dir* | Defines a directory path. Interacting with a plug of this type opens a directory browser dialog.|`String`|
 | *Enumerator* | Defines a list of selectable values. It is not possible to create custom Enumerator plugs in a node.|`String`|
 | *FileIn* | Defines an input file. This type is used for open and import operations. Interacting with a plug of this type opens a file browser dialog to select an existing file.|`String`|
-| *FileOut* | Defines an output file. This type is used for save and export operations. Interacting with a plug of this type opens a file browser dialog, although it is possible to enter a not existing file. Selecting an existing file will override the file the after execution. |`String`|
+| *FileOut* | Defines an output file. This type is used for save and export operations. Interacting with a plug of this type opens a file browser dialog, although it is possible to enter a non-existent file. Selecting an existing file will override the file after the workflow executes. |`String`|

```

Comment:
```suggestion
| *FileOut* | Defines an output file. This type is used for save and export operations. Interacting with a plug of this type  in the inspector opens a file browser dialog. Selecting an existing file will override the file after the workflow executes, , alternatively it is possible to enter a non-existent file name to create it. |`String`|

```

Reply:
Done. I rewrote it a bit.
https://github.com/Inbibo/shift_docs/pull/71/commits/532d1cecf1db648a2d7d5506dbe4ed982d776fbb